spec/schemas/_common.yaml

```diff
@@ -152,9 +152,9 @@ components:
       x-version-added: '2.4'
       description: |-
         A two-dimensional Cartesian point specified by x and y coordinates. It can be represented in the following ways:
-        - As a `{x, y}` object
-        - As a `[x, y]` array
-        - As a string in `"x, y"` or Well Known Text (WKT) point formats
+         - As a `{x, y}` object.
```

Comment:
These should not have periods.

Reply:
We need to use periods at the end of description these other why the spec will error out. Would periods for the other points work?

Reply:
Ah, OK, then let's definitely leave them.